packages/MSBot/src/msbot-disconnect.ts

```diff
@@ -8,7 +8,7 @@ import * as chalk from 'chalk';
 import * as program from 'commander';
 
 program.Command.prototype.unknownOption = (flag: string): void => {
-    console.error(chalk.default.redBright(`Unknown arguments: ${flag}`));
+    console.error(chalk.default.redBright(`[msbot] Unknown arguments: ${flag}`));
```

Comment:
>msbot [](start = 44, length = 5)

Why are we making this change at all?

Reply:
@tomlm - MSBOT is a critical tool that will be used in a lot of piped input processing scenarios with LUIS, QnA Maker, Dispatch and possibly others in the future. 

So when piping inputs, its really hard to tell what tool is reporting the error. This is especially true when you try something like luis import application --msbot | msbot connect luis --stdin and you have bad input to LUIS, or if you had a bad .bot file, its hard to tell from just the error as to what tool is reporting this. 

Starting with MSBOT, I believe we should do another PR to have LUIS, QnA Maker and Dispatch as well include a prefix on all errors so it is clear on which tool is reporting the error.